core/src/main/proto/spotify_100.proto

```diff
@@ -12,23 +12,18 @@ message Metric {
     // time in ms when metric was generated.
     int64 time = 2;
 
-    // value of metric.
```

Comment:
Based on the [docs](https://developers.google.com/protocol-buffers/docs/proto#reserved) we shouldn't remove the fields but mark them **reserved**